api/schemas/locationInfo.yaml

```diff
@@ -0,0 +1,33 @@
+# {
+#  "timestamp-ms": 1585906080346,
+#  "userId": 34753543,
+#  "anonymous":false,
+#  "inside": true,
+#  "placeIds": [
+#    453785436,
+#    71984147938
+#  ]
+# }
+type: object
+description: Specifics of an user's current location relative to a place.
+required:
+  - timestampMs
+  - userId
+  - anonymous
+  - inside
+  - placeIds
+# TODO add properties description and examples?
+properties:
+  timestampMs:
+    type: integer
+    format: uint64
```

Comment:
Perchè il timestamp qua è in millisecondi, mentre in appConnections è una stringa in formato date-time?